### PR TITLE
OFT Wrapper integration

### PR DESCRIPTION
**Pull request by satoshi-lifi**

### CODE REVIEW

**Comment by satoshi-lifi** [comment]
Deployed and tested OFTWrapperFacet on Goerli testnet

- OFT
   Sending: https://goerli.etherscan.io/tx/0xedb4503ea3f5fbd07e109896d934ade574ca56a27c9ca8b473cf5c1d445d060d
   Receiving: https://testnet.bscscan.com/tx/0x529f6378849245395f4916572eccfe97e5ad4481c0b35134cbe40cf1eca5fa18
- OFTV2
   Sending: https://goerli.etherscan.io/tx/0xd05ef00c4db284927dccacf16bf4e3a538e3f7358d1d859609d6048fb3ee7c97
   Receiving: https://testnet.bscscan.com/tx/0xe11256ffab4e0dcd1eb0d9c2ca2487392e1fbe4a954191ad1a35b06e6de43f98
- OFTFeeV2
   Sending: https://goerli.etherscan.io/tx/0x3b147dc5e6be5954b302ce27d0c571e556e3b45c8f16aac238f5e0e887188fe9
   Receiving: https://testnet.bscscan.com/tx/0xa5fd6d74538187264d5ecd612ec3c3581b3cbf0a62bb3156707d3a48057c221f

File: script/DeployOFTWrapperFacet.s.sol [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        string memory path = string.concat(
+            vm.projectRoot(),
+            "/config/oftwrapper.json"
+        );
+        string memory json = vm.readFile(path);
+        address oftWrapper = json.readAddress(
+            string.concat(".wrappers.", network)
+        );
+
+        if (oftWrapper == address(32)) {
```
Comment (0xDEnYO) [inline script/DeployOFTWrapperFacet.s.sol:26]: why are we comparing here to address(32) and not to address(0) ???

**Review by 0xDEnYO — APPROVED** [review]

File: script/DeployOFTWrapperFacet.s.sol (hunk 1 above)
Comment (0xDEnYO) [reply script/DeployOFTWrapperFacet.s.sol:26]: @satoshi-lifi 
would be nice if you could still respond to this although the PR has been merged already. I was a bit fast with the merging :)

Comment (satoshi-lifi) [reply script/DeployOFTWrapperFacet.s.sol:26]: when the element doesn't exist in the json, the value is `address(32)` but not `address(0)`

**Pull request merged**